invenio_records_lom/services/schemas/metadata.py

```diff
@@ -18,50 +18,63 @@ class Meta:
 
         unknown = INCLUDE
 
+    def dump(self, obj, **_):
+        """Overwrite dump to return `obj`, bypassing validation."""
```

Comment:
Git-diff cuts off at an unlucky place here:
The schema this method appears in is called [`NoValidationSchema`](https://github.com/tu-graz-library/invenio-records-lom/blob/b3bbf2bbc37e8030a8abdc2b66ed84df6d20158f/invenio_records_lom/services/schemas/metadata.py#L13).
The implications of not validating within something called `NoValidationSchema` should be obvious enough I hope?

P.S. For the record: Without overwriting dump, the following case behaves differently:
```python
# dump `{"a": "0"}`, `NoValidationSchema.dump` should return it as is
Schema().dump({"a": "0"})  # code as previously, uses marshmallow.Schema.dump (from its parent class), returns `{}`
DumpOverwrittenSchema().dump({"a": "0"})  # code as of now, returns `{"a": "0"}`
# i.e. `"a"` isn't included in the dump, despite the configuration `unknown = marshmallow.INCLUDE`
```

P.P.S. should actually be `copy.copy(obj)` for consistency with marshmallow

Reply:
yes, it seams obvious, but it would help to add a small note to the `def dump` docstring like. `Overwrite dump to return obj, bypassing validation as class name indicates`. don't put the burden of collecting information to the reader, if you could help them that easily.

Reply:
Done.